### ISSUE TITLE
Implement `avdata_import()`

### ISSUE THREAD

**Issue by mtmorgan** [Bioconductor/AnVIL#44]
Using `Terra()$importAttributesTSV`? See https://support.bioconductor.org/p/9141388/

**Comment by smgogarten** [comment]
I'd like to replicate the Workspace Data table that I can read in from this public example, but in a new workspace:
```
avdata(namespace="amp-t2d-op", name="2019_ASHG_Reproducible_GWAS-V2")
```

**Comment by mtmorgan** [comment]
Sorry for the delay; this should be implemented in version 1.7.5, available in the devel branch (only) via `BiocManager::install("Bioconductor/AnVIL")` now or via `BiocManager::install("AnVIL")` once the package builds and checks successfully.

The solution is not perfect. It does not yet handle the 'Description' column. Table rows are either updated or appended, but not deleted. A value that appears as a vector of hyperlinks in the UI round-trips (to R and back to AnVIL as a JSON array-of-characters `["gs://...", "gs://...", ...]`. The first two of these will be addressed in a separate issue.

For what it's worth, https://github.com/Bioconductor/AnVIL/issues/45 may address the issue @smgogarten reported separately about an authentication error; please let me know if you continue to see authentication errors...
